lib/board.rb

```diff
@@ -0,0 +1,56 @@
+require 'pry'
+class Board
+
+  attr_accessor :cells
+
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    #binding.pry
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(int)
+    int = int.to_i
```

Comment:
we can just remove line 25 and change line 26 to be 
```ruby 
cells[int.to_i - 1]
```

I would also look at changing int to be input all over your code. It looks more readable